lib/uname.js

```diff
@@ -0,0 +1,43 @@
+var getUname = require('node-uname')
+var decodeArguments = require('./common').decodeArguments
+
+module.exports = uname
+
+function uname(args) {
+    var params = decodeArguments(args)
+    var info = getUname()
+
+    if (params.a || params.all) {
+        return [info.sysname, info.nodename, info.release, info.version, info.machine].join(' ')
```

Comment:
Not exactly, what I'm asking for is that you do something similar to

```Javascript
var info = getUname()
info.inspect = function()
{
  // Implement `inspect()` here
}
return info
```

You can take examples from other commands on this library, for example grep.